MonoGame.Framework/Graphics/SpriteFont.cs

```diff
@@ -83,7 +106,7 @@ public int GetHashCode(char b)
         /// <remarks>Can be used to calculate character bounds when implementing custom SpriteFont rendering.</remarks>
         public Dictionary<char, Glyph> GetGlyphs()
```

Comment:
Random thought here.

Maybe we should start to depreciate this method and add a new type: `GlyphDictionary`.  It would provide a read only `Dictionary` like API, but uses your fast region lookup internally.  This can then be used directly by `SpriteFont` and `SpriteBatch` as well as user's custom font measuring/rendering.